app/models/debate_outcome.rb

```diff
@@ -22,4 +40,28 @@ def date
   def debate_state
     debated? ? 'debated' : 'none'
   end
+
+  def image_ratio(width, height)
+    (width.to_f / height.to_f).round(2)
+  end
+
+  def validate_commons_image_dimensions
+    # This should be tuned if the images start looking badly scaled
+    max_ratio_delta = 0.1
+
+    dimensions = Paperclip::Geometry.from_file(commons_image.queued_for_write[:original].path)
+
+    # Too big:
+    errors.add(:commons_image, "Width must be at least #{COMMONS_IMAGE_SIZE[:w]}px (is #{dimensions.width}px)") unless dimensions.width >= COMMONS_IMAGE_SIZE[:w]
+    errors.add(:commons_image, "Height must be #{COMMONS_IMAGE_SIZE[:h]}px (is #{dimensions.height}px)") unless dimensions.height >= COMMONS_IMAGE_SIZE[:h]
```

Comment:
These lines are too long for trailing conditionals and the errors should be passed as symbols and then looked up from the locale file like other Active Record errors.